### PR TITLE
Added devhub proposal notification

### PR DESCRIPTION
**Pull request by Megha-Dev-19**

### CODE REVIEW

**Comment by Megha-Dev-19** [comment]
@frol I have added `devhub/like` and `devhub/reply` to UI, so that the user is notified correctly about proposal (instead of devhub.near account), let me know if there something else you expect here.

File: src/NearOrg/Notifications/utils.jsx [hunk 1]
```diff
@@ -105,6 +114,17 @@ function createNotificationLink(notificationType, notificationValue, authorAccou
...
+    case "devhub/mention":
+    case "devhub/edit":
+    case "devhub/reply":
+    case "devhub/like":
+      return href({
+        widgetSrc: "devhub.near/widget/app",
+        params: {
+          page: "proposal",
+          id: proposal,
+        },
```
Comment (frol) [inline src/NearOrg/Notifications/utils.jsx:126]: @Megha-Dev-19 I think you also need to pass the [`notifier` to the parameters](https://github.com/NEAR-DevHub/neardevhub-contract/commit/46ca5c3f2b469e1a56b5773ae6224928b51a1649)

Comment (Megha-Dev-19) [reply src/NearOrg/Notifications/utils.jsx]: Oh I didn't push the commit, just did, update the profile as well

**Review by frol — APPROVED** [review]

**Comment by frol** [comment]
@charleslavon Can you help with the review or assigning someone to review and merge/release it?

**Comment by shelegdmitriy** [comment]
LGTM! Seems like there are two new fields which comes from `value`: `proposal` and `notifier`. Both of them has `string` type, right? I'm asking this because we [released](https://github.com/near/near-discovery-components/pull/642) usage of `notification` indexer for addition page. And seems this two fields are missed there and I'm happy to help with this! Also, we'll need to update our service worker. All I need from you is to confirm that the types of new fields are correct so I could create a task to update `notifications` indexer.

cc @frol @Megha-Dev-19

**Review by shelegdmitriy — APPROVED** [review]

**Comment by Megha-Dev-19** [comment]
@shelegdmitriy hello, you're right about notifier, but for proposal it will be an integer.

**Comment by shelegdmitriy** [comment]
> proposal

Thanks @Megha-Dev-19! I created an issue for this. May I ask you to provide me some accountId which can this new notification type for testing please?

**Comment by Megha-Dev-19** [comment]
@shelegdmitriy yes you can use `devhub.near`.

**Comment by shelegdmitriy** [comment]
Thank you @Megha-Dev-19 for your contribution!

**Pull request merged**